src/groups/mwc/mwcsys/mwcsys_executil.t.cpp

```diff
@@ -130,7 +130,7 @@ static void test2_executeSystemFailure()
 
     PVV("Testing abnormal exit of the command");
     rc = mwcsys::ExecUtil::execute(&output,
-                                   "python -c 'import os,signal; "
+                                   "python3 -c 'import os,signal; "
```

Comment:
We cannot use implicit `python2` here, `python3` alias might work on most frequently known systems

Reply:
I'm fine with this test.  If it fails, we can use use `python3.11` etc, but I'd rather not tie ourselves to a specific Python version if we can help it.